src/App/Pages/ContactPage.elm

```diff
@@ -10,8 +10,18 @@ import Material.Options as Options
 import Material.Button as Button
 
 
-view : Model -> Html Msg
-view model =
+init : ContactForm
+init =
+    { name = ""
+    , email = ""
+    , telephone = ""
+    , subject = ""
+    , message = ""
+    }
+
+
+view : Model -> ContactForm -> Html Msg
```

Comment:
not in this PR, but the `ContactM` name is really not clear. You probably mean something like [this](https://github.com/Gizra/drupal-elm-starter/blob/f93745bfcab3e3d28a38ff20838d97482aed1eea/client/src/elm/App/Model.elm#L24)?